### PR TITLE
fix some typos

### PR DESCRIPTION
**Pull request by matthyx**

### CODE REVIEW

File: docs/blog/posts/introducing-kubescape-3/introducing-kubescape-3.md [hunk 1]
```diff
@@ -201,4 +201,4 @@ The Kubescape project follows the [CNCF Code of Conduct](https://github.com/cncf
 
 ## Watch this space
 
-We will be publishing more detailed posts about the new features in Kubescape 3.0 in the coming weeks. Stay tuned!
\ No newline at end of file
+We will be publishing more detailed posts about the new features in Kubescape 3.0 in the coming weeks. Stay tuned!
```
Comment (Oshratn) [inline docs/blog/posts/introducing-kubescape-3/introducing-kubescape-3.md:204]: This is really outdated. Maybe get rid of the whole thing.

File: docs/project/community.md [hunk 2]
```diff
@@ -15,7 +15,7 @@ We have three mailing lists:
 * [-dev](https://lists.cncf.io/g/cncf-kubescape-dev): for discussion amongst contributors
 * [-users](https://lists.cncf.io/g/cncf-kubescape-dev): for user-to-user discussion and help
 
-We recommend all Kubescape users join -announce to stay up to date with new releases.
+We recommend all Kubescape users join -announce to stay up-to-date with new releases.
 
 ## Monthly meeting
```
Comment (Oshratn) [inline docs/project/community.md:20]: No longer a monthly meeting. It's bi-weekly.

**Comment by matthyx** [comment]
@Oshratn I have corrected the 2 notes

**Comment by matthyx** [comment]
@Oshratn rebased for conflicts

File: docs/docs/guides/kubescape-gha/index.md [hunk 3]
```diff
@@ -63,7 +63,7 @@ First, create a new branch called dev:
 
 `git checkout -b dev`
 
-We’re going to do a very simple manifest that will deploy an Nginx pod to start. Although simple, it still contains several security vulnerabilities, as you can see. Create a file called nginx.yml in the root directory of your repository and add the following:
+We’re going to do a very simple manifest that will deploy a Nginx pod to start. Although simple, it still contains several security vulnerabilities, as you can see. Create a file called nginx.yml in the root directory of your repository and add the following:
```
Comment (Oshratn) [inline docs/docs/guides/kubescape-gha/index.md:66]: I've seen NGINX or nginx (on their docs) but never mixed case. 
Please select according to your preference.

**Review by Oshratn — DISMISSED** [review]

File: docs/docs/guides/kubescape-gha/index.md (hunk 3 above)
Comment (matthyx) [reply docs/docs/guides/kubescape-gha/index.md]: done

**Review by Oshratn — APPROVED** [review]

**Pull request merged**